tests/user_commands/tests.py

```diff
@@ -42,6 +42,11 @@ def test_command_style(self):
         management.call_command('dance', '--style', 'Jive', stdout=out)
         self.assertIn("I don't feel like dancing Jive.\n", out.getvalue())
 
+    def test_command_outputs_newline(self):
+        out = StringIO()
+        management.call_command('dance', "--newline", stdout=out, verbosity=0)
+        self.assertEqual("\n", out.getvalue())
```

Comment:
I don't think that we need a separate test for this PR. It's already tested implicitly for builtin commands, so I've added this to an existing test.

Reply:
Thank you!